src/content/overview/fleet-management/cluster-management/cluster-concepts/releases/index.md

```diff
@@ -23,7 +23,93 @@ user_questions:
+
+Old workload cluster releases get [deprecated](#deprecation) when replaced by newer ones. After deprecation, and once no longer in use by any customer, a workload cluster release gets archived, which means that it is no longer accessible to you.
+
+### Deprecation {#deprecation}
+
+Once we provide a new workload cluster release, be it major, minor, or patch, we usually deprecate an older version.
+
+- As soon as we publish a new major release, we can deprecate all active releases in the two major versions preceding it, as none of them are needed as an upgrade path anymore.
+- When releasing a new minor or patch version, we deprecate the previous version. For example, when 28.1.0 comes out, the previous version 28.0.1 gets deprecated.
+
+Once deprecated, you can still continue to use the workload cluster release with existing clusters. In addition, as long as you have at least one cluster in your installation running the deprecated release, you can also create new clusters using that release. This allows you to create test clusters to test an upgrade from that release.
+
+### Pre-releases
+
+Pre-releases (when available) provide you with the option to test new features or use new functionality before the feature is officially released. At the same time, it helps us release new features and test them in a real-world scenario with your help. When you test a pre-release, it helps identify bugs or assess the impact of new functionality on the system.
+
+Currently not all Giant Swarm releases become available as pre-releases. We communicate pre-releases, when they are available for customer testing.
+
+Pre-releases can be alpha, beta, or release candidates (RC). This depends on the needs of the team wanting to test their feature.
+
+Use Cases for Pre-Releases:
+
+- A new feature with unclear impact is done and the team wants to test and evaluate its impact before deciding the scope of the release (alpha). Customers get early access to the functionality and see if it satisfies their requirements. Customers report back with bugs and additional commentary. This allows the functionality to be evaluated in a less controlled way than in our testing environments. It also identifies bugs that we may have not found, allowing us to fix them early. It also may bring about reconsidering the feature/solution in general.
+
+- A new high impact feature is done, but needs to wait for the next major release (beta). Customers get early access to the functionality and see that it sits well with their set-up. Customers report back with bugs as necessary. This allows us to make fixes to bugs that we may have not found.
+
+- A new high-impact feature is done and is urgently needed by a customer (release candidate `rc`). The customer can get a pre-release to test and roll out ASAP. This release is complete and stable. It may not meet the quality bars of a general availability (GA) release.
```

Comment:
```suggestion
- A new high-impact feature is done and is urgently needed by a customer (release candidate `rc`). The customer can get a pre-release to test and roll out as soon as possible. This release is complete and stable. It may not meet the quality bars of a general availability (GA) release.
```